templates/journal/story.html

```diff
@@ -7,14 +7,35 @@
   <div id="story-panel" class="panel-outer">
     <p class="panel-heading-title">{{ story.title.upper() }}</p>
     <p class="panel-heading-author"><a target="_blank" href="{{ story.author.url }}">{{ story.author }}</a></p>
-    <div class="panel-inner">
-      <div class="panel-content">
-        {{ story.content | safe }}
-      </div>
-    </div>
+       <div>
+           <div class="mobile-audio-player" id="mobile">
+                <iframe style="border: 0; width: 100%; height: 42px" src="https://bandcamp.com/EmbeddedPlayer/album={{issue.bandcamp_album_id}}/size=small/bgcol=ffffff/linkcol=0687f5/track={{story.bandcamp_track_id}}/transparent=true/" seamless><a href="{{issue.bandcamp_album_url}}">Sponge: {{issue.pretty_name}}</a></iframe>
+            </div>
+            <div class="panel-inner">
+              <div class="panel-content">
+                {{ story.content | safe }}
+              </div>
+            </div>
+       </div>
   </div>
   <div id="contents-actions-container">
-    <div>
+     {% if story.bandcamp_track_id %}
+            {% set height_of_bandcamp_plugin = [142] %}
+                {% for story in stories_in_issue %}
+                    {% if story.bandcamp_track_id %}
+                        {% if height_of_bandcamp_plugin.append(height_of_bandcamp_plugin.pop() + 33) %}
+                        {% endif %}
+                    {% endif %}
+                {% endfor %}
+            {% set height_of_bandcamp_plugin = height_of_bandcamp_plugin.pop()| string + 'px' %}
+            <div style="height:calc({{height_of_bandcamp_plugin}} + 1rem + 4px)" class="tab">
+                <button class="active" id="contents-tab" onclick="openTab('contents')"><span>Contents</span></button>
```

Comment:
This is good, but the text is hard to centre and to make look good. I reckon icons could look better.